src/Bridge/Doctrine/Orm/Filter/RangeFilter.php

```diff
@@ -102,7 +101,7 @@ private function addWhere(QueryBuilder $queryBuilder, QueryNameGeneratorInterfac
                 $rangeValue = explode('..', $value);
 
                 if (2 !== count($rangeValue)) {
```

Comment:
Would be best then removing it imo
On Mon, 5 Sep 2016 at 16:47, Teoh Han Hui <notifications@github.com> wrote:

> In src/Bridge/Doctrine/Orm/Filter/RangeFilter.php
> <https://github.com/api-platform/core/pull/723#discussion_r77532077>:
>
> > @@ -102,7 +101,7 @@ private function addWhere(QueryBuilder $queryBuilder, QueryNameGeneratorInterfac
> >                  $rangeValue = explode('..', $value);
> >
> >                  if (2 !== count($rangeValue)) {
>
> Yes, we could just log them at the "notice" level, probably.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/api-platform/core/pull/723/files/4cb9ae04642ca8c91dd503fbf6d920273f04cb77#r77532077>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABQr89YBWOt9DPA6pZzcK7nuARmKNx-rks5qnCtqgaJpZM4J1Ard>
> .
>